kde-apps/libkmahjongg/libkmahjongg24-24.12.3.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="Mahjongg library based on Qt/KDE Frameworks"
+DESCRIPTION="This directory contains the library used for loading and rendering of \
+Mahjongg tilesets and associated backgrounds, used by KMahjongg and KShisen."
+HOMEPAGE="https://invent.kde.org/games/libkmahjongg"
+COPYRIGHT="2010-2024 KDE Organisation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/libkmahjongg-$portVersion.tar.xz"
+CHECKSUM_SHA256="db2bdb5b8214f28ad68fdac424bec23fb1862121b729982c8af26b0b0e13f6c6"
+SOURCE_DIR=libkmahjongg-$portVersion
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="6.0.241203"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libkmahjongg24$secondaryArchSuffix = $portVersion
```

Comment:
Please rename to `libkmahjongg_kf6` (also the recipe), I've dropped suffix 24 (25 is on it's way), this reflects better the target it was intended for.

Reply:
PS, if you intend to update kmahjongg and kshisen also, maybe just drop the whole suffix here and drop the one for 23.08.5.